ios/safari_extension/NIC/control.pl

```diff
@@ -0,0 +1,5 @@
+NIC->variable("PARENTPROJECTNAME") = "";
+
+while(NIC->variable("PARENTPROJECTNAME") eq "") {
+	NIC->variable("PARENTPROJECTNAME") = NIC->prompt("PROJ", "Name of parent project (required)");
```

Comment:
It may be worth specifying that by parent project name we mean an application name